extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -119,7 +119,8 @@ message PluginConfig {
   // (Optional) Collection of tag names and tag expressions to include in the
   // logs. Conflicts are resolved by the tag name by overriding previously
   // supplied values. Does not apply to audit logs.
-  // See https://istio.io/latest/docs/tasks/observability/metrics/customize-metrics/#use-expressions-for-values
+  // See
+  // https://istio.io/latest/docs/tasks/observability/metrics/customize-metrics/#use-expressions-for-values
```

Comment:
Is it possible to use versioned url?

Reply:
We'd need to keep updating it with every release. I don't expect much to change. Not sure if modules should be versioned with istio versions.